stable/gcloud-sqlproxy/README.md

```diff
@@ -25,7 +25,7 @@ You need to enable Cloud SQL Administration API and create a service account for
 Install from remote URL with the release name `pg-sqlproxy` into namespace `sqlproxy`, set GCP service account and SQL instances and ports:
 
 ```console
-$ helm upgrade pg-sqlproxy rimusz/gcloud-sqlproxy --namespace sqlproxy \
+$ helm install pg-sqlproxy rimusz/gcloud-sqlproxy --namespace sqlproxy \
```

Comment:
it makes more sense to change to `$ helm upgrade --install pg-sqlproxy ...` which works for the first install and following upgrades

Reply:
oh well I just knew that we can use "upgrade" for install also 👍